### PR TITLE
steering: Reflect results of 2021 election

### PR DESCRIPTION
**Pull request by justaugustus**
2022 Steering Committee:

- Ashley Wolf - @ashleywolf, GitHub
- Dawn Foster - @geekygirldawn, VMware
- Shilla Saebi - @shillasaebi, Comcast
- Stephen Augustus - @justaugustus, Cisco
- Stephen Jacobs - @itprofjacobs, RIT
- Thomas Steenbergen - @tsteenbe, HERE Technologies
- VM (Vicky) Brasseur - @vmbrasseur, Wipro

Signed-off-by: Stephen Augustus <foo@auggie.dev>

Thank you, @caniszczyk, @duaneobrien, @stormypeters, @kpfleming!

cc: @anajsana 
ref: https://github.com/todogroup/governance/issues/151

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,10 +10,10 @@ Our mission is to identify key policy and process choices related to corporate o
...
 
-* Chris Aniszczyk (@caniszczyk) - (8/15/2019 -  12/31/2021)
-* Duane O'Brien (@duaneobrien) - (8/15/2019 - 12/31/2021)
-* Stormy Peters (@stormypeters - (1/1/2021 - 12/31/2022)
-* Kevin P. Fleming (@kpfleming) (8/15/2019 - 12/31/2021)
-* Dawn Foster (@geekygirldawn) (1/1/2021- 12/31/2022)
-* Stephen Jacobs (@itprofjacobs) - Appointed, (1/1/2021 - 12/31/2022)
-* Ashley Wolf (@ashleywolf)- Appointed, (6/1/2020 - 12/31/2021)
+- [Ashley Wolf](https://github.com/ashleywolf), GitHub
+- [Dawn Foster](https://github.com/geekygirldawn), VMWare
```
Comment (geekygirldawn) [inline README.md:14]: Again, this should be VMware :)

File: README.md [hunk 2]
```diff
@@ -10,10 +10,10 @@ Our mission is to identify key policy and process choices related to corporate o
...
+- [Shilla Saebi](https://github.com/shillasaebi), Comcast
+- [Stephen Augustus](https://github.com/justaugustus), Cisco
+- [Stephen Jacobs](https://github.com/itprofjacobs), RIT
+- [Thomas Steenbergen](https://github.com/tsteenbe), HERE Technologies
+- [VM (Vicky) Brasseur](https://github.com/vmbrasseur), Wipro
```
Comment (geekygirldawn) [inline README.md:19]: Can we add the dates back in? It is super helpful when preparing for the next election to know when people's terms end.

Comment (geekygirldawn) [reply README.md:19]: We also need to know which seats were appointed, so can you add that data back into the table?

**Review by geekygirldawn — CHANGES_REQUESTED** [review]
Again, we should wait to merge this until after Jan 1.

File: README.md (hunk 1 above)
Comment (justaugustus) [reply README.md]: Fixed!

File: README.md (hunk 2 above)
Comment (justaugustus) [reply README.md]: > Can we add the dates back in? It is super helpful when preparing for the next election to know when people's terms end.
> We also need to know which seats were appointed, so can you add that data back into the table?

Fixed! Added a table (that includes elected vs appointed) sorted by start date, then end date, then first name.

Comment (justaugustus) [reply README.md]: (someone please check my math on the terms :))

**Comment by justaugustus** [comment]
@ashleywolf -- Integrated your suggestions from caeb793ad9209e08486a4b4b5a5fc6e3f6c6ce40 in https://github.com/todogroup/governance/pull/152/commits/cb91e2d0f3770cc5d13c83819508758ceb6489a2 (and attributed you).

**Pull request merged**